Create App Service plan was wrong

kemura opened the pull request:
Hi, 
My customer found that create app service plan command was wrong. There was  "--is-linux" option in spite this document is for Windows. 
I fixed but am not sure my fixing is right. Would you confirm what I changed ?

azurecla commented:
Hi __@rjmax__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ryjones). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tfitzmac commented:
This looks good to me. It can be merged.

tfitzmac commented:
@davidwrede this can be merged.

Commit:
Create App Service plan was wrong 

Hi, 
My customer found that create app service plan command was wrong. There was  "--is-linux" option in spite this document is for Windows. 
I fixed but am not sure my fixing is right. Would you confirm what I changed ?
